src/services/studio/studio.service.ts

```diff
@@ -52,6 +106,50 @@ class StudioService {
             } : pkg;
         });
     }
+
+    private deleteFileIfTypeScenario(packageZip: AdmZip, entry: IZipEntry): void {
+        if (entry.entryName.startsWith("nodes/") && entry.entryName.endsWith(".yml")) {
+            const node: NodeExportTransport = parse(entry.getData().toString());
+            if (node.type === "SCENARIO") {
+                packageZip.deleteFile(entry);
+            }
+        }
+    }
+
+    private fixConnectionVariablesIfRootNodeFile(packageZip: AdmZip, entry: IZipEntry, zipName: string, exportedVariables: VariableManifestTransport[]): void {
+        if (entry.name === "package.yml") {
+            const packageKeyAndVersion = zipName.replace(".zip", "").split("_");
+            const connectionVariablesByKey = this.getConnectionVariablesByKeyForPackage(packageKeyAndVersion[0], packageKeyAndVersion[1], exportedVariables);
+
+            if (connectionVariablesByKey.size) {
+                const exportedNode: NodeExportTransport = parse(entry.getData().toString());
+                const nodeContent: NodeSerializedContent = parse(exportedNode.serializedContent);
+
+                nodeContent.variables = nodeContent.variables.map(variable => ({
+                    ...variable,
+                    metadata: variable.type === PackageManagerVariableType.CONNECTION ? {
+                        ...variable.metadata,
+                        ...connectionVariablesByKey.get(variable.key).metadata
```

Comment:
I changed this to set the metadata to the `connectionVariablesByKey` since we are already fixing by appending to these variables metadata [here](https://github.com/celonis/content-cli/pull/167/files#diff-10b781e4b0f7b649247dbfc2845e42dd87ef2b6ea9f130b055c3675655151d2dR36).